sysdig/internal/client/v2/group_mapping_config.go

```diff
@@ -0,0 +1,105 @@
+package v2
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+)
+
+var GroupMappingConfigNotFound = errors.New("group mapping configuration not found")
+
+const (
+	CreateGroupMappingConfigPath = "%s/api/groupmappings/settings"
```

Comment:
We have here only Get and Put requests on the same URL path. Lets just have one variable `GroupMappingConfigPath="%s/api/groupmappings/settings"`

I would also remove `CreateGroupMappingConfig` since its doing exactly the same thing as update. I would just keep `GetGroupMappingConfig` and `UpdateGroupMappingConfig`